cognite/client/utils/_time.py

```diff
@@ -60,6 +60,10 @@ def _import_zoneinfo_not_found_error() -> type[ZoneInfoNotFoundError]:
     return ZoneInfoNotFoundError
 
 
+def _datetime_is_tz_unaware(dt: datetime) -> bool:
+    return dt.tzinfo is None or dt.tzinfo.utcoffset(dt) is None
```

Comment:
Why the last part? It is still aware if it is set to UTC imo
```suggestion
def _datetime_is_tz_unaware(dt: datetime) -> bool:
    return dt.tzinfo is None
```

Reply:
I agree!